js/widgets/accordion.js

```diff
@@ -31,7 +31,19 @@
 
 $.widget( "ui.accordion", $.ui.accordion, {
 	options: {
-		theme: null
+		theme: null,
+		icons: {
+			activeHeader: "ui-icon-caret-d",
+			header: "ui-icon-caret-r"
+		}
+	},
```

Comment:
Do I understand it correctly that if we would use ```visibility: hidden;``` instead of ```display: none;``` we wouldn't need this JS?
If so, maybe we should prioritize https://github.com/jquery/jquery-mobile/issues/8137 and see if we can make that change in 1.5.